aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -174,6 +174,458 @@ static inline __host__ __device__ scalar_t calc_polygamma(int n, scalar_t x) {
   return ((n % 2) ? 1.0 : -1.0) * ::exp(::lgamma(static_cast<scalar_t>(n) + 1.0)) * zeta(static_cast<scalar_t>(n + 1), x);
 }
 
+// regularized lower & upper incomplete gamma
+template <typename scalar_t>
```

Comment:
Are these functions completely duplicative with the CPU header updates?

Reply:
Yes ...